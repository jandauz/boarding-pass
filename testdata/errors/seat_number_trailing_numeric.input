M1DESMARAIS/LUC       EABC123 YULFRAAC 0834 326J00100025 100