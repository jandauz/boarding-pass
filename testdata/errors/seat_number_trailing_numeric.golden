bcbp: Invalid data format:
  boarding pass data:
  | "M1DESMARAIS/LUC       EABC123 YULFRAAC 0834 326J00100025 100"
  |                                                  ^ got "0010"
  |
  = reason: data for "Seat Number" must be 3 digits with leading zeroes followed by an alpha
